browser/ipfs/test/BUILD.gn

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import("//brave/build/config.gni")
+import("//testing/test.gni")
+
+source_set("unittests") {
+  testonly = true
+  sources = [
+    "//brave/browser/ipfs/content_browser_client_helper_unittest.cc",
+    "//brave/browser/ipfs/ipfs_host_resolver_unittest.cc",
+    "//brave/browser/ipfs/test/ipfs_navigation_throttle_unittest.cc",
+    "//brave/browser/net/ipfs_redirect_network_delegate_helper_unittest.cc",
+  ]
+
+  deps = [
+    "//base/test:test_support",
+    "//brave/browser/net:net",
+    "//brave/browser/tor:tor",
+    "//brave/browser:browser_process",
+    "//brave/components/ipfs",
+    "//brave/components/tor/buildflags:buildflags",
```

Comment:
If target name is the same as the folder, we should just put `"//brave/components/tor/buildflags"`